src/dotnet-archive/dotnet-archive.csproj

```diff
@@ -6,7 +6,7 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
     <EnableApiCheck>false</EnableApiCheck>
-    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
Any reason this isn't win-x64? We are using the updated consolidated RIDs elsewhere in our build.